### ISSUE TITLE
How to make multiple verbs in the same url return different files?

### ISSUE THREAD

**Issue by tammorelli** [gstroup/apimocker#19]
Hello, I am trying to mock a get/post using something like this:

"webServices": {
        "user/:userId": {
            "latency": 700,
            "verbs": ["get", "post"],
            "responses": {
                "get": {
                    "mockFile": "user.json"
                },
                "post": {
                    "mockFile": "user_save.json"
                }
            }
        },
        ...

But I am receiving the following warning from grunt:

WARNING: apimocker config file format is deprecated.
Warning: undefined is not a function Use --force to continue.

Aborted due to warnings.

It only happens when I try using the "responses" object, if I remove it and use a single "mockFile" it works fine.

**Comment by gstroup** [comment]
Hi,

This is a bug that I can fix pretty easily.  I'm surprised we haven't caught this one earlier!  I will try to fix this in the next day.  For now, you can put in another "mockFile" entry in your config.json, like this "foo.json":
```
{
    "user/: userId": {
        "mockFile": "foo.json",
        "latency": 700,
        "verbs": [
            "get",
            "post"
        ],
        "responses": {
            "get": {
                "mockFile": "user.json"
            },
            "post": {
                "mockFile": "user_save.json"
            }
        }
    }
}
```

**Comment by gstroup** [comment]
Fixed in version 0.4.0.
